cypress/e2e/hermioneAccount.spec.js

```diff
@@ -1,11 +1,98 @@
 /// <reference types='cypress' />
+import { faker } from '@faker-js/faker';
 
 describe('Bank app', () => {
-  before(() => {
-    cy.visit('/');
+  beforeEach(() => {
+    cy.visit('');
   });
 
+  const userHermiona = 'Hermoine Granger';
+  const accounNumber = 1001;
+  const currency = 'Dollar';
+  const amountPlaceholder = 'amount';
+  const depositAmount = `${faker.number.int({ min: 1648, max: 2023 })}`;
+  const buttonDeposit = 'Deposit';
+  const depositMessage = 'Deposit Successful';
+  const buttonWithdrawl = 'Withdrawl';
+  const withdrawlAmount = `${faker.number.int({ min: 1648, max: depositAmount })}`;
+  const withdrawlMassage = 'Transaction successful';
+  const buttonTransactions = 'Transactions';
+  const changeAccountNumber = 1003;
+
   it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.get('.btn.btn-primary.btn-lg').should('be.visible');
+    cy.contains('.btn.btn-primary.btn-lg', 'Customer Login').click();
+
+    cy.get('#userSelect').select(userHermiona);
+    cy.get('button[type="submit"]').click();
+
+    cy.contains('#accountSelect', accounNumber).should('be.visible');
+
+    cy.get('.center').should('contain.text', 'Account Numb');
+    cy.contains('.ng-binding', accounNumber).should('be.visible');
```

Comment:
The variable `accounNumber` is used here, but it seems to have a typo. Consider using `accountNumber` instead.